_po/ja/how-to/laravel.po

```diff
@@ -337,59 +496,81 @@ msgid ""
 "    {\n"
 "        $posts = \\App\\Models\\Post::query()->paginate(10);"
 msgstr ""
+"class PostController extends Controller\n"
+"{\n"
+"    public function index() : View\n"
+"    {\n"
+"        $posts = \\App\\Models\\Post::query()->paginate(10);"
 
 msgid ""
 "        return view('posts.index', compact('posts'));\n"
 "    }"
 msgstr ""
+"        return view('posts.index', compact('posts'));\n"
+"    }"
 
 msgid ""
 "    public function search(Request $request) : View\n"
 "    {\n"
 "        $keyword = $request->input('keyword');"
 msgstr ""
+"    public function search(Request $request) : View\n"
+"    {\n"
+"        $keyword = $request->input('keyword');"
 
 msgid ""
 "        $posts = \\App\\Models\\Post::search($keyword)->paginate(10)->withQuerySt"
 "ring();"
 msgstr ""
+"        $posts = \\App\\Models\\Post::search($keyword)->paginate(10)->withQuerySt"
+"ring();"
 
 msgid ""
 "        return view('posts.index', compact('posts', 'keyword'));\n"
 "    }"
 msgstr ""
+"        return view('posts.index', compact('posts', 'keyword'));\n"
+"    }"
 
 msgid ""
 "}\n"
 "```"
 msgstr ""
+"}\n"
+"```"
 
 msgid "### Routing"
-msgstr ""
+msgstr "### ルート設定"
 
 msgid "We will also need to create URLs for our controller."
-msgstr ""
+msgstr "コントローラで取り扱うURLを作成します。"
 
 msgid "To start with, we are going to enable two routes:"
-msgstr ""
+msgstr "ここでは対応する2つのルートを有効にします。"
 
 msgid ""
 "* The `index` route will display our listing of blog posts.\n"
 "* The `search` route will be used for searching blog posts."
 msgstr ""
+"* `index`ルートはブログ記事一覧を表示させます。\n"
+"* `search`ルートはブログ記事を検索するのに利用されます。"
 
 msgid "Edit `routes/web.php` like this:"
-msgstr ""
+msgstr "`routes/web.php`を次のように編集します:"
 
 msgid ""
 "```php\n"
 "<?php"
 msgstr ""
+"```php\n"
+"<?php"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.